code/modules/paperwork/pen.dm

```diff
@@ -62,6 +64,25 @@
 	user << "<span class='notice'>\The [src] will now write in [colour].</span>"
 	desc = "It's a fancy four-color ink pen, set to [colour]."
 
+
+/obj/item/weapon/pen/attack_self(mob/living/carbon/user)
+	var/deg = input(user, "What angle would you like to rotate the pen head to? (1-360)", "Rotate Pen Head") as null|num
+	if(deg && (deg > 0 && deg <= 360))
+		degrees = deg

```

Comment:
because then a number the user didn't want to put in, would be put in.
Yes they'd be told of this, but I feel it'd be better for them to learn the lesson that, Yes, it really does only mean 1-360, as opposed to "lol I can put 500 in and I still get a value out"